sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -956,7 +961,18 @@ object Radar {
                             return
                         }
 
-                        verificationManager.getIntegrityToken(config.googlePlayProjectNumber, config.nonce) { integrityToken, integrityException ->
+                        if (config.nonce == null) {
+                            logger.e("Error missing nonce for trackVerified", RadarLogType.SDK_ERROR)
```

Comment:
@jsani-radar @tderouin Today I think we only use `SDK_ERROR` for exceptions. I think we can just make this a debug message instead